### PR TITLE
Convert GetterSetter to typescript

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#169639179](https://www.pivotaltracker.com/story/show/169639179/)

### CODE REVIEW

File: evm/v0.5/testv2/GetterSetter.test.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      await gs.setBytes32(bytes32)
+
+      const currentBytes32 = await gs.getBytes32()
+      assert.deepEqual(h.toUtf8(currentBytes32), h.toUtf8(bytes32))
+    })
+
+    it('logs an event', async () => {
+      const tx = await gs.connect(roles.stranger).setBytes32(bytes32)
+      const receipt = await tx.wait()
+      const args: any = receipt.events?.[0].args
```
Comment (j16r) [inline evm/v0.5/testv2/GetterSetter.test.ts]: Isn't `args: any` kind of redundant?

Comment (NavyAdmiral) [reply evm/v0.5/testv2/GetterSetter.test.ts]: @j16r It's actually not, because from the generated typings, `.args` is `any[]`

Comment (j16r) [reply evm/v0.5/testv2/GetterSetter.test.ts]: Isn't `any` the "world" type, i.e. `any[]` is a member of `any`? i.e. what happens if you don't put `any` here?

Comment (NavyAdmiral) [reply evm/v0.5/testv2/GetterSetter.test.ts]: Line 51 will complain with `Property 'from' does not exist on type 'any[]'.`

**Pull request merged**